### ISSUE TITLE
Producing diff in calledWith

### ISSUE THREAD

**Issue by hellboy81** [sinonjs/sinon#773]
``` js
AssertionError: expected update to have been called with arguments {
...
}, {...}

functionName({...}, {,,,})
```

It is not easy to see difference.

Please provide diffs with red/green

**Comment by hdh3000** [comment]
@hellboy81 can you give a few more tangible examples? think i may have a fix, but its a bit unclear what you are asking for..

**Comment by jcrben** [comment]
You can draw some inspiration from a similar example with mocha. I found https://github.com/litixsoft/karma-mocha-reporter/issues/32 which helped get mocha's colored diffing into the karma-mocha-reporter.

You can see what it looks like over at https://github.com/litixsoft/karma-mocha-reporter#showdiff

**Comment by jcrben** [comment]
fyi, just posted a $15 bounty for this https://www.bountysource.com/issues/23366130-producing-diff-in-calledwith

**Comment by fatso83** [comment]
http://unexpected.js.org might be worth a look.

**Comment by KaidenR** [comment]
It looks like this may be duplicated (and fixed) in #1179

**Comment by fatso83** [comment]
Fixed in sinon@next
